src/node.h

```diff
@@ -200,6 +200,7 @@ NODE_EXTERN extern bool force_fips_crypto;
 # endif
 #endif
 
+NODE_EXTERN void RegisterBuiltinModules();
```

Comment:
Can you add a short doc comment for this method, in particular that embedders need to call it on startup?

I know a lot in this file isn’t very well-documented, but at least new code should come with something. :)